nilearn/_utils/fixes/scikit_learn_validation.py

```diff
@@ -0,0 +1,400 @@
+# scikit-learn v0.17
+
+import numpy as np
+import warnings
+import scipy.sparse as sp
+
+
+class NotFittedError(ValueError, AttributeError):
```

Comment:
No, this is imported from another software. Backports must be left as untouched as possible. Otherwise updating them (say with code from sklearn 0.18 for example when it will be released) can be tedious.